docs/general-usage-instructions.html

```diff
@@ -0,0 +1,21 @@
+<!--Installation and configuration instructions. HTML tags are allowed.-->
+
+<ol>
+  <li>Install the plugin from <b>Marketplace</b></li>
+  <li>
+    Open Git Machete tab:
+    <ul>
+      <li><b>Git</b> (tool window) -> <b>Git Machete</b> (tab)</li>
+      <li>Use key shortcut: Ctrl + Alt + Shift + M</li>
```

Comment:
This unordered list looks more like **both** actions are necessary rather than just **any** of them :thinking: